rhcos4/profiles/stig.profile

```diff
@@ -0,0 +1,23 @@
+documentation_complete: true
+
+title: '[DRAFT] DISA STIG for Red Hat Enterprise Linux CoreOS'
+
+description: |-
+    This profile contains configuration checks that align to the
+    [DRAFT] DISA STIG for Red Hat Enterprise Linux CoreOS which
+    is the operating system layer of Red Hat OpenShift Container
+    Platform.
+
+extends: ospp
+
+selections:
+    - login_banner_text=dod_banners
+    - banner_etc_issue
+    - audit_rules_usergroup_modification_passwd
+    - sssd_enable_smartcards
+    - sssd_offline_cred_expiration
+    - encrypt_partitions
+    - accounts_tmout
+    - sudo_remove_no_authenticate
+    - sudo_remove_nopasswd
+    - sudo_require_authentication
```

Comment:
By default the core user is a member of the sudo group which has `%sudo        ALL=(ALL)       NOPASSWD: ALL` set in sudoers, this likely clash with the sudo rules above. Just saying.

Reply:
Right, however it is a problem to just sudo without password. Also, I believe that the recommended guidance that we are going to also give is to remove the core user.